.github/workflows/service_test_redis.yml

```diff
@@ -142,6 +110,125 @@ jobs:
           OPENDAL_REDIS_ROOT: /
           OPENDAL_REDIS_DB: 0
 
+  redis-cluster:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Rust toolchain
+        uses: ./.github/actions/setup
+        with:
+          need-nextest: true
+      - name: Configure Redis Cluster
```

Comment:
Apologies for bringing up CI-related matters, but would you be interested in maintaining a docker-compose file similar to the one found at https://github.com/bitnami/containers/blob/main/bitnami/redis-cluster/docker-compose.yml? This could make local testing easier for us.

If you're not keen on this aspect of the work, we can proceed with merging this PR first. I'm willing to help migrate this after.

Reply:
tracked at https://github.com/apache/incubator-opendal/issues/2860

Reply:
> Apologies for bringing up CI-related matters, but would you be interested in maintaining a docker-compose file similar to the one found at https://github.com/bitnami/containers/blob/main/bitnami/redis-cluster/docker-compose.yml? This could make local testing easier for us.
> 
> If you're not keen on this aspect of the work, we can proceed with merging this PR first. I'm willing to help migrate this after.

Sure, but I'm not sure which directory to put the `docker-compose.yml` file in.